test/test_quantized_deconvolutional_layer.h

```diff
@@ -13,74 +13,71 @@
+    EXPECT_EQ(l.in_data_shape().size(), serial_size_t(1));   // number of inputs shapes
     EXPECT_EQ(l.out_data_shape().size(), serial_size_t(1));  // num of output shapes
-    EXPECT_EQ(l.weights().size(),        serial_size_t(2));  // the wieghts vector size
-    EXPECT_EQ(l.weights_grads().size(),  serial_size_t(2));  // the wieghts vector size
-    EXPECT_EQ(l.inputs().size(),         serial_size_t(3));  // num of input edges
-    EXPECT_EQ(l.outputs().size(),        serial_size_t(2));  // num of outpus edges
-    EXPECT_EQ(l.in_types().size(),       serial_size_t(3));  // num of input data types
-    EXPECT_EQ(l.out_types().size(),      serial_size_t(2));  // num of output data types
-    EXPECT_EQ(l.fan_in_size(),           serial_size_t(9));  // num of incoming connections
-    EXPECT_EQ(l.fan_out_size(),          serial_size_t(18)); // num of outgoing connections
-    EXPECT_STREQ(l.layer_type().c_str(), "q_deconv");      // string with layer type
+    EXPECT_EQ(l.weights().size(), serial_size_t(2));         // the wieghts vector size
+    EXPECT_EQ(l.weights_grads().size(), serial_size_t(2));   // the wieghts vector size
+    EXPECT_EQ(l.inputs().size(), serial_size_t(3));          // num of input edges
+    EXPECT_EQ(l.outputs().size(), serial_size_t(2));         // num of outpus edges
+    EXPECT_EQ(l.in_types().size(), serial_size_t(3));        // num of input data types
+    EXPECT_EQ(l.out_types().size(), serial_size_t(2));       // num of output data types
+    EXPECT_EQ(l.fan_in_size(), serial_size_t(9));            // num of incoming connections
+    EXPECT_EQ(l.fan_out_size(), serial_size_t(18));          // num of outgoing connections
+    EXPECT_STREQ(l.layer_type().c_str(), "q_deconv");        // string with layer type
 }
 
 TEST(quantized_deconvolutional, fprop) {
     typedef network<sequential> CNN;
-    CNN nn;
+    CNN                         nn;
 
     quantized_deconvolutional_layer<sigmoid> l(2, 2, 3, 1, 2);
 
-    // layer::forward_propagation expects tensors, even if we feed only one input at a time
-    auto create_simple_tensor = [](size_t vector_size) {
-        return tensor_t(1, vec_t(vector_size));
-    };
+    // layer::forward_propagation expects tensors, even if we feed only one input
+    // at a time
+    auto create_simple_tensor = [](size_t vector_size) { return tensor_t(1, vec_t(vector_size)); };
 
     // create simple tensors that wrap the payload vectors of the correct size
-    tensor_t in_tensor     = create_simple_tensor(4)
-           , out_tensor    = create_simple_tensor(32)
-           , a_tensor      = create_simple_tensor(32)
-           , weight_tensor = create_simple_tensor(18)
-           , bias_tensor   = create_simple_tensor(2);
+    tensor_t in_tensor = create_simple_tensor(4), out_tensor = create_simple_tensor(32),
+             a_tensor = create_simple_tensor(32), weight_tensor = create_simple_tensor(18),
+             bias_tensor = create_simple_tensor(2);
```

Comment:
Also, positions of spaces are aligned manually in original source code.
It looks like clang-format isn't ordered to respect the style.